CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Current version
 
+### Bug fixes
+
+* [Issue 220](https://github.com/MassimoCimmino/pygfunction/issues/220) - Fixed the expected line length in `boreholes.field_from_file()` to correctly import fields of inclined boreholes.
+
```

Comment:
I'm not sure how you want to handle the change log in the maintenance branch.

Reply:
This is fine. The header will be changed to the bugfix version in a separate issue and PR.